### PR TITLE
/pos/touch product list pagination #1308

### PR DESCRIPTION
**Pull request by ithiame**

### CODE REVIEW

File: src/routes/(app)/pos/touch/+page.svelte [hunk 1]
```diff
@@ -32,11 +36,29 @@
...
 						<ProductWidgetPOS {product} pictures={picturesByProduct[product._id]} />
 					{/if}
 				{/each}
+				<div class="col-span-2 grid-cols-1 flex gap-2 justify-center">
+					{#if next > 0}
+						<a
+							class="btn touchScreen-product-secondaryCTA"
+							on:click={() => (next = Math.max(0, next - POS_PRODUCT_PAGINATION))}
+							href={`?filter=${filter}&skip=${Math.max(0, next - POS_PRODUCT_PAGINATION)}`}
+							>&#9664;</a
```
Comment (coyotte508) [inline src/routes/(app)/pos/touch/+page.svelte:50]: Let's use an emoji, unicode character, `&lt;` or even an imported icon but not a number qualifier

File: src/routes/(app)/pos/touch/+page.svelte [hunk 2]
```diff
@@ -32,11 +36,29 @@
...
+						>
+					{/if}
+					PAGE {currentPage}/{totalPages}
+					{#if next + POS_PRODUCT_PAGINATION < productFiltered.length}
+						<a
+							class="btn touchScreen-product-secondaryCTA"
+							on:click={() => (next += POS_PRODUCT_PAGINATION)}
+							href={`?filter=${filter}&skip=${next + POS_PRODUCT_PAGINATION}`}>&#9654;</a
```
Comment (coyotte508) [inline src/routes/(app)/pos/touch/+page.svelte:58]: Same

**Review by coyotte508 — APPROVED** [review]
ok with changes

**Comment by Tirodem** [comment]
Please make the next / previous button more visible, I've barely saw them in the current version.

Then, change the active page to 1 when the user change tag / product list, otherwise, we can have this kind of situations :
image
image

**Pull request merged**